controllers/utils/node_roles.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import v1 "k8s.io/api/core/v1"
+
+const (
+	// WorkerRoleLabel is the role label of worker nodes
+	WorkerRoleLabel = "node-role.kubernetes.io/worker"
+	// MasterRoleLabel is the old role label of control plane nodes
+	MasterRoleLabel = "node-role.kubernetes.io/master"
+	// ControlPlaneRoleLabel is the new old role label of control plane nodes
```

Comment:
NIT:
Why "new old role label" and not  "new role label" ("new old role label" -> "new role label")?

Reply:
because it's a typo 🤣 
thx, fixed